src/relentless/simulate/hoomd.py

```diff
@@ -49,13 +50,6 @@ class Flags(enum.IntEnum):
 else:
     _gsd_write_mode = "wb"
 
-try:
```

Comment:
I'm good with making freud a required import above, but you should follow the pattern for the other packages of making a `_freud_version` variable here that we can access below:
```
_freud_version = packaging.version.parse(freud.__version__)
```